modules/ROOT/pages/bolt/message.adoc

```diff
@@ -742,16 +757,57 @@ Valid values for `api` and the corresponding API associations are:
 - `2` -- Implicit transaction
 - `3` -- Driver-level `execute_query()`
 
+*Detail messages:*
+
+No detail messages should be returned.
+
+*Valid summary messages:*
+
+** `SUCCESS`
+** `FAILURE`
+
 ==== Synopsis
 ----
-TELEMETRY {metadata}
+TELEMETRY api
+----
+
+.Example 1
+----
+TELEMETRY 0
+----
+
+==== Server response `SUCCESS`
+
+If a `TELEMETRY` message request has been successfully received, the server should respond with a `SUCCESS` and remain in its current state.
```

Comment:
```suggestion
If a `TELEMETRY` message request is successfully received, the server responds with a `SUCCESS` and remains in its current state.
```
No `should` because it is a behavior that is out of the control of the client, and readers are not asked to implement behaviors on the server side. We can assume it _always_ happens what it _should_ happen.

Reply:
Fair point. I copy pasted this from another place. Wanna go over the whole docs and fix it as well? :)

Reply:
Can't just now, but at some point I will :)